src/NzbDrone.Api/Series/SeriesModule.cs

```diff
@@ -113,14 +112,19 @@ private List<SeriesResource> AllSeries()
 
         private int AddSeries(SeriesResource seriesResource)
         {
-            var series = _seriesService.AddSeries(seriesResource.InjectTo<Core.Tv.Series>());
+            var model = seriesResource.ToModel();
 
-            return series.Id;
+#warning TODO: Fetch from skyhook in the service before creating? (In AddSeries)
```

Comment:
Fetch the details of the series, that'd be nice to reduce what the client _needs_ to send (especially for 3rd party integration).